scss/_patterns_navigation.scss

```diff
@@ -296,14 +362,41 @@ $navigation-hover-opacity: 0.58;
     }
   }
 
-  @include deprecate('3.0.0', 'Selectors should use classes.') {
-    .p-navigation__link.is-selected > a {
-      opacity: $navigation-hover-opacity;
-    }
+  %navigation-link-dark-selected {
+    opacity: $navigation-hover-opacity;
   }
 
-  // separator color on small screens
-  .p-navigation__link > a::before {
+  %navigation-link-dark-before {
+    // separator color on small screens
     background: $colors--dark-theme--border-default;
   }
+
+  .p-navigation__item > .p-navigation__link, // remove nesting in 3.0 when deprecated __link styles are removed
+  .p-navigation__toggle--close,
+  .p-navigation__toggle--open {
+    @extend %navigation-link-dark;
+  }
+
+  .p-navigation__item.is-selected > .p-navigation__link {
```

Comment:
Do we need the sibling selector now that we are directly targeting classes?

Reply:
Yes. In this case it depends on `is-selected` state of the parent. I guess we could simplify it to `.p-navigation__item.is-selected .p-navigation__link` if that's what you mean.